drm/DrmAtomicStateManager.cpp

```diff
@@ -73,7 +73,7 @@ auto DrmAtomicStateManager::CommitFrame(AtomicCommitArgs &args) -> int {
     return -ENOMEM;
   }
 
-  int out_fence = -1;
+  int out_fence = 0;
```

Comment:
Can you double check how this change fixed any issue?
Is the below function actually expecting the address of a local(stack) variable as a uint64_t ?

Reply:
The issue found was: "out_fence" is passed to a parameter that cannot be negative -> for line 212
Initializing it to 0 resolves the issue.